src/components/Gallery/styled.ts

```diff
@@ -112,13 +113,17 @@ export const CarouselButton = styled(GalleryButton as any)`
 export const ImageWrapper = styled.div`
   text-align: center;
   width: 100%;
-  height: 100%;
-  display: inline-block;
+  height: auto;
+  display: flex;
+  flex-direction: column;
   transition-property: transform, visibility;
-
+  font-size: 0;
+  margin: auto;
   img {
     object-fit: contain;
-    height: 100%;
+    height: auto;
+    min-height: 25vh;
+    max-height: 75vh;
```

Comment:
is this new logic of min and max height rules?

Reply:
These seemed logical to me.  Otherwise the gallery could be bigger than a phone screen, and when no image was available (broken image, no imageUrl), it would shrink down to 1rem.  Always open to modification.

Reply:
I feel like the min- and max-height almost solves this ticket alone